<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurePerGame</class>
 <widget class="QDialog" name="ConfigurePerGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>607</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="templateStyleSheet" stdset="0">
   <string>QDialog {
    background-color: #2b2b2b;
    color: #ffffff;
    }

    QWidget {
    background-color: #2b2b2b;
    color: #ffffff;
    }

    QStackedWidget {
    background-color: #2b2b2b;
    border: 1px solid #3d3d3d;
    border-radius: 8px;
    margin: 0px;
    padding: 0px;
    }

    QScrollArea {
    background-color: #2b2b2b;
    border: none;
    border-radius: 8px;
    }

    QScrollArea > QWidget > QWidget {
    background-color: #2b2b2b;
    }

    QScrollBar:vertical {
    background-color: #3d3d3d;
    width: 14px;
    border-radius: 7px;
    margin: 2px;
    }

    QScrollBar::handle:vertical {
    background-color: #5d5d5d;
    border-radius: 6px;
    min-height: 30px;
    margin: 1px;
    }

    QScrollBar::handle:vertical:hover {
    background-color: %%ACCENT_COLOR%%;
    }

    QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: none;
    background: none;
    height: 0px;
    }

    QScrollBar:horizontal {
    background-color: #3d3d3d;
    height: 14px;
    border-radius: 7px;
    margin: 2px;
    }

    QScrollBar::handle:horizontal {
    background-color: #5d5d5d;
    border-radius: 6px;
    min-width: 30px;
    margin: 1px;
    }

    QScrollBar::handle:horizontal:hover {
    background-color: %%ACCENT_COLOR%%;
    }

    QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    border: none;
    background: none;
    width: 0px;
    }

    QPushButton.tabButton {
    background-color: #383838;
    color: #ffffff;
    padding: 10px 14px;
    margin: 2px;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
    border-bottom-left-radius: 8px;
    border-bottom-right-radius: 8px;
    min-width: 85px;
    max-width: 160px;
    font-weight: 500;
    border: 1px solid #3d3d3d;
    text-align: center;
    }

    QPushButton.tabButton:checked {
    background-color: %%ACCENT_COLOR%%;
    color: #ffffff;
    font-weight: bold;
    border-color: %%ACCENT_COLOR%%;
    }

    QPushButton.tabButton:hover:!checked {
    background-color: #4d4d4d;
    border-color: #5d5d5d;
    }

    QPushButton.tabButton:pressed {
    background-color: %%ACCENT_COLOR_PRESSED%%;
    }

    QTabWidget {
    background-color: #2b2b2b;
    border: none;
    }

    QTabWidget::pane {
    border: 1px solid #3d3d3d;
    background-color: #2b2b2b;
    border-radius: 8px;
    margin: 0px;
    padding: 0px;
    }

    QTabWidget::tab-bar {
    alignment: left;
    }

    QTabBar {
    background-color: #2b2b2b;
    border: none;
    }

    QTabBar::tab {
    background-color: #383838;
    color: #ffffff;
    padding: 8px 14px;
    margin-right: 2px;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
    min-width: 85px;
    font-weight: 500;
    border: 1px solid #3d3d3d;
    border-bottom: none;
    }

    QTabBar::tab:selected {
    background-color: %%ACCENT_COLOR%%;
    color: #ffffff;
    font-weight: bold;
    border-color: %%ACCENT_COLOR%%;
    }

    QTabBar::tab:hover:!selected {
    background-color: #4d4d4d;
    border-color: #5d5d5d;
    }

    QTabBar QToolButton {
    background-color: #383838;
    border: 1px solid #3d3d3d;
    border-radius: 4px;
    padding: 4px;
    margin: 2px;
    }

    QTabBar QToolButton:hover {
    background-color: #4d4d4d;
    border-color: %%ACCENT_COLOR%%;
    }

    QTabBar::scroller {
    width: 30px;
    }

    QGroupBox {
    font-weight: bold;
    border: 1px solid #3d3d3d;
    border-radius: 8px;
    margin-top: 12px;
    padding-top: 12px;
    background-color: #2b2b2b;
    color: #ffffff;
    }

    QGroupBox::title {
    subcontrol-origin: margin;
    left: 12px;
    padding: 0 8px 0 8px;
    color: #ffffff;
    font-weight: bold;
    }

    QCheckBox {
    color: #ffffff;
    spacing: 10px;
    padding: 4px;
    background-color: transparent;
    }

    QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border: 2px solid #5d5d5d;
    border-radius: 4px;
    background-color: #3d3d3d;
    }

    QCheckBox::indicator:checked {
    background-color: %%ACCENT_COLOR%%;
    border-color: %%ACCENT_COLOR%%;
    }

    QCheckBox::indicator:hover {
    border-color: %%ACCENT_COLOR%%;
    }

    QComboBox {
    background-color: #3d3d3d;
    border: 1px solid #5d5d5d;
    border-radius: 6px;
    padding: 8px 12px;
    color: #ffffff;
    min-width: 120px;
    min-height: 28px;
    selection-background-color: %%ACCENT_COLOR%%;
    }

    QComboBox:hover {
    border-color: %%ACCENT_COLOR%%;
    background-color: #404040;
    }

    QComboBox:focus {
    border-color: %%ACCENT_COLOR%%;
    background-color: #404040;
    }

    QComboBox::drop-down {
    border: none;
    width: 25px;
    subcontrol-origin: padding;
    subcontrol-position: top right;
    background-color: transparent;
    }

    QComboBox QAbstractItemView {
    background-color: #3d3d3d;
    border: 1px solid %%ACCENT_COLOR%%;
    selection-background-color: %%ACCENT_COLOR%%;
    color: #ffffff;
    outline: none;
    }

    QComboBox QAbstractItemView::item {
    padding: 8px;
    border: none;
    background-color: transparent;
    }

    QComboBox QAbstractItemView::item:selected {
    background-color: %%ACCENT_COLOR%%;
    color: #ffffff;
    }

    QComboBox QAbstractItemView::item:hover {
    background-color: %%ACCENT_COLOR_HOVER%%;
    color: #ffffff;
    }

    QLineEdit {
    background-color: #3d3d3d;
    border: 1px solid #5d5d5d;
    border-radius: 6px;
    padding: 8px 12px;
    color: #ffffff;
    min-height: 20px;
    selection-background-color: %%ACCENT_COLOR%%;
    }

    QLineEdit:focus {
    border-color: %%ACCENT_COLOR%%;
    background-color: #404040;
    }

    QPushButton {
    background-color: %%ACCENT_COLOR%%;
    color: #ffffff;
    border: none;
    padding: 10px 20px;
    border-radius: 6px;
    font-weight: bold;
    min-height: 20px;
    }

    QPushButton:hover {
    background-color: %%ACCENT_COLOR_HOVER%%;
    }

    QPushButton:pressed {
    background-color: %%ACCENT_COLOR_PRESSED%%;
    }

    QPushButton:disabled {
    background-color: #5d5d5d;
    color: #8d8d8d;
    }

    QToolButton {
    background-color: %%ACCENT_COLOR%%;
    color: #ffffff;
    border: none;
    padding: 8px 12px;
    border-radius: 6px;
    font-weight: bold;
    min-width: 32px;
    min-height: 24px;
    }

    QToolButton:hover {
    background-color: %%ACCENT_COLOR_HOVER%%;
    }

    QToolButton:pressed {
    background-color: %%ACCENT_COLOR_PRESSED%%;
    }

    QLabel {
    color: #ffffff;
    background-color: transparent;
    padding: 2px;
    }

    QListWidget {
    background-color: #3d3d3d;
    border: 1px solid #5d5d5d;
    border-radius: 6px;
    color: #ffffff;
    padding: 4px;
    }

    QListWidget::item {
    padding: 8px;
    border-radius: 4px;
    margin: 1px;
    }

    QListWidget::item:selected {
    background-color: %%ACCENT_COLOR%%;
    color: #ffffff;
    }

    QListWidget::item:hover:!selected {
    background-color: #4d4d4d;
    }

    QSlider::groove:horizontal {
    border: 1px solid #5d5d5d;
    height: 8px;
    background-color: #3d3d3d;
    border-radius: 4px;
    }

    QSlider::handle:horizontal {
    background-color: %%ACCENT_COLOR%%;
    border: 1px solid %%ACCENT_COLOR%%;
    width: 18px;
    margin: -5px 0;
    border-radius: 9px;
    }

    QSlider::handle:horizontal:hover {
    background-color: %%ACCENT_COLOR_HOVER%%;
    }

    QSpinBox, QDoubleSpinBox {
    background-color: #3d3d3d;
    border: 1px solid #5d5d5d;
    border-radius: 6px;
    padding: 6px;
    color: #ffffff;
    min-height: 20px;
    }

    QSpinBox:focus, QDoubleSpinBox:focus {
    border-color: %%ACCENT_COLOR%%;
    background-color: #404040;
    }

    QRadioButton {
    color: #ffffff;
    spacing: 8px;
    padding: 4px;
    }

    QRadioButton::indicator {
    width: 16px;
    height: 16px;
    border: 2px solid #5d5d5d;
    border-radius: 8px;
    background-color: #3d3d3d;
    }

    QRadioButton::indicator:checked {
    background-color: %%ACCENT_COLOR%%;
    border-color: %%ACCENT_COLOR%%;
    }

    QRadioButton::indicator:hover {
    border-color: %%ACCENT_COLOR%%;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>318</width>
          <height>538</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Info</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item alignment="Qt::AlignHCenter">
             <widget class="QGraphicsView" name="icon_view">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>256</width>
                <height>256</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>256</width>
                <height>256</height>
               </size>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="interactive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="6" column="1">
               <widget class="QLineEdit" name="display_size">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="display_version">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Title ID</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="display_title_id">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QLineEdit" name="display_filename">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="display_format">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Filename</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="display_name">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="display_developer">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Format</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Version</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Size</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Developer</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_build_id">
                <property name="text">
                 <string>Base Build ID</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QLineEdit" name="display_build_id">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_update_build_id">
                <property name="text">
                 <string>Update Build ID</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QLineEdit" name="display_update_build_id">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VerticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="usesScrollButtons">
          <bool>true</bool>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Some settings are only available when a game is not running.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurePerGame</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurePerGame</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
